index.html

```diff
@@ -20,7 +20,29 @@
 			</div>
 		</header>
 		<main id="main"></main>
-		<footer></footer>
+		<footer>
```

Comment:
The footer should have the class .footer too
```suggestion
		<footer class="footer">
```